pkg/controller/master/upgradelog/common.go

```diff
@@ -416,6 +401,23 @@ func prepareLogDownloaderSvc(upgradeLog *harvesterv1.UpgradeLog) *corev1.Service
 	}
 }
 
+// Returns the name of the log-archive PVC, which is created by the fluentd StatefulSet.
+//
+//	The name will look like: <upgradelog>-infra-log-archive-<fluentd statefulset>-0
```

Comment:
Is `<fluentd statefulset>-0` equal to `<upgradelog>-infra-fluentd-0`?

Reply:
The name is kind of a mess. Yes you're right. Let me update it to match the code.